webdriver-ts/src/isKeyed.ts

```diff
@@ -196,35 +221,53 @@ async function assertClassesContained(elem: ElementHandle<HTMLElement>, expected
 
 export async function checkTRcorrect(page: Page): Promise<boolean> {
   let tr = await page.$("tbody>tr:nth-of-type(1000)");
-  if (!(await assertChildNodes(tr as ElementHandle<HTMLElement>, ["td", "td", "a", "td", "a", "span", "td"], "tr"))) {
+  if (
+    !(await assertChildNodes(
+      tr as ElementHandle<HTMLElement>,
+      ["td", "td", "a", "td", "a", "span", "td"],
+      "tr"
+    ))
+  ) {
     return false;
   }
 
   // first td
   let td1 = await page.$("tbody>tr:nth-of-type(1000)>td:nth-of-type(1)");
-  if (!(await assertClassesContained(td1 as ElementHandle<HTMLElement>, ["col-md-1"], "first td"))) {
+  if (
```

Comment:
That "improvement" really looks silly...

Reply:
Yeah, it looks like shit.